README.md

```diff
@@ -429,6 +429,20 @@ The default values for each rule are described below.
 | duplicate_sibling_description | warning |
 | incorrect_ref_pattern        | warning |
 
+## Warnings Limit
+
+To impose a warnings limit on a project, add a `.thresholdrc` to your project. It is recommended to add this file to the root of the project. The validator recursively searches up the filesystem from whichever directory the validator is invoked, and the nearest `.thresholdrc` will be used.
```

Comment:
I think there could be a little more explanation as to what the limit actually means - that if the amount of warnings exceeds the limit, the exit code will be non-zero and CI builds would fail

Reply:
Added: "You may impose a warning limit on your API definitions. If the number of warnings issued exceeds the warning limit, the **exit code will be set to 1**."